src/model.rs

```diff
@@ -729,8 +729,22 @@ pub trait ModelWithProblem {
     /// Returns whether the constraint should be separated during LP processing
     fn cons_is_separated(&self, cons: &Constraint) -> bool;
 
-    /// Writes the optimization model to a file with the given path and extension.
-    fn write(&self, path: &str, ext: &str) -> Result<(), Retcode>;
+    /// Write the problem to a file using SCIP's writer
+    ///
+    /// # Arguments
+    ///
+    /// * `path` - The path to the file (without extension).
+    /// * `ext` - The file extension (e.g., "lp", "mps").
+    /// * `symb` - If true, use symbolic names given by user for variables and constraints; if false, use indices given by SCIP.
+    ///
+    /// # Returns
+    ///
+    /// * `Result<(), Retcode>` - Ok(()) if successful, Err(Retcode) otherwise.
+    ///
+    /// # Notes
+    ///
+    /// * Linked to test in `write_and_read_lp`
```

Comment:
```suggestion
```

I think we don't need this part, users looking for how it's used in tests can search for the function name. In general it is good to not have things in comments that could be outdated later when something else changes (like the test's name in this case).

You can apply the suggested change from this comment no need to push yourself.